invesalius/data/viewer_slice.py

```diff
@@ -565,17 +565,31 @@ def SetBrushColour(self, colour):
         if self.slice_data.cursor:
             self.slice_data.cursor.SetColour(colour_vtk)
 
-    def UpdateSlicesNavigation(self, arg, position):
+    def UpdateSlices(self, position):
```

Comment:
I think this name UpdateSlices is not good. I think the name doesn't say what this method does.

Reply:
Agreed. What could be better? Because from what I understood this method uses the ScrollSlice to change the scroll position and thus "update" the view on all slices after the cross or the scroll has moved.. could it be `UpdateSlicesView` ?

Reply:
I was thinking in something Like UpdateSlicesPosition.

Reply:
Sounds great.